script.js

```diff
@@ -1,66 +1,151 @@
 let content;
-window.onload = () => { content = document.getElementById("content"); }
+window.onload = () => {content = document.getElementById("content"); loadFromLocalStorage();}
 const DONE_TASK_TEXT_CLASS = "task-input-text-corssed";
 
-function addEmptyTask() {
-    let task = createEmptyTask();
-    content.appendChild(task);
-}
+class Task{
+    constructor(){
+        this.div = document.createElement("div");
```

Comment:
Task properties should be `text` and `isChecked`, the HTML element will be served by `.toHtmlElement()` function

Reply:
Task properties should be `text` and `isChecked`, the HTML element will be served by `.toHtmlElement()` function

Reply:
done